docs/pages/release_notes.md

```diff
@@ -50,6 +50,10 @@ The remaining section describe the complete release notes for 7.0.0.
   in order to have the same meaning as in 6.55.0. That will make it easier if you upgrade from 6.55.0 to 7.0.0.
   However, that means, that you need to change these method calls if you have migrated to 7.0.0-rc1 already.
 
+#### Rule Changes
+* {% rule "java/design/ImmutableField" %}: the property `ignoredAnnotations` has been removed. The property was
+  deprecated since PMD 6.52.0.
```

Comment:
I'd include here a note as to how to move around this through `violationSuppressXPath` with a small sample such as:

>  If you were previously doing
> 
>  ```xml
>    <property name="ignoredAnnotations" value="java.lang.Deprecated" />
>  ```
> 
>  you can instead use
>  ```xml
>   <property name="violationSuppressXPath" value=".[not(pmd-java:hasAnnotation('java.lang.Deprecated'))" />
>  ```


Disclaimer: I've not tested the `violationSuppressXPath` is correct :P